ecosystem/python/sdk/aptos_sdk/client.py

```diff
@@ -162,11 +162,9 @@ def transaction_pending(self, txn_hash: str) -> bool:
     def wait_for_transaction(self, txn_hash: str) -> None:
         """Waits up to 10 seconds for a transaction to move past pending state."""
 
-        count = 0
+        count_start = time.time()
         while self.transaction_pending(txn_hash):
-            assert count < 10, f"transaction {txn_hash} timed out"
-            time.sleep(1)
-            count += 1
+            assert count_start + 10 > time.time(), f"transaction {txn_hash} timed out"
```

Comment:
this will hammer the rest server, I did some analysis on devnet and found that it only saved a fraction of a second. So if you really want fine grained, I'd support sleep for .1 or .25.

Reply:
okay. we still need to have some delay by default here -- if you want to make this configurable during client initialization, I'd be happy to accept that, but this as a default is not.

Reply:
> okay. we still need to have some delay by default here -- if you want to make this configurable during client initialization, I'd be happy to accept that, but this as a default is not.

Configurable? Where should fit the hole? And no config file here... unless you want me to write on the configurable global varable on client.py...